lib/src/driver/driver.dart

```diff
@@ -120,6 +120,10 @@ class BazelWorkerDriver {
           _readyWorkers.remove(worker);
           _runWorkQueue();
         });
+      }).catchError((Object e, StackTrace? s) {
```

Comment:
[nit] `onError` has a better typed argument

```suggestion
      }).onError((e, s) {
```

`s` is also non-nullable this way, but I guess it didn't matter in the body.

Reply:
I also went ahead and just provided the string 'null' as the default error if it is null, 🤷‍♂️

Reply:
Hmm, I expected `e` to also be non-nullable here. But maybe that would require `onError<Object>(`?

Reply:
Ah yeah, I can do that. If we get nulls here I am OK with that being weird probably.